src/libraries/System.Net.Security/src/System/Net/Security/NegotiateAuthentication.cs

```diff
@@ -356,21 +356,23 @@ public NegotiateAuthenticationStatusCode UnwrapInPlace(Span<byte> input, out int
         /// Computes the message integrity check of a given message.
         /// </summary>
         /// <param name="message">Input message for MIC calculation.</param>
-        /// <param name="signature">Buffer writer where the MIC is written.</param>
+        /// <param name="signatureWriter">Buffer writer where the MIC is written.</param>
         /// <remarks>
+        /// Implements the GSSAPI GetMIC operation.
+        ///
         /// The method modifies the internal state and may update sequence numbers depending on the
         /// selected algorithm. Two successive invocations thus don't produce the same result and
         /// it's important to carefully pair GetMIC and VerifyMIC calls on the both sides of the
         /// authenticated session.
         /// </remarks>
-        internal void GetMIC(ReadOnlySpan<byte> message, IBufferWriter<byte> signature)
+        public void ComputeIntegrityCheck(ReadOnlySpan<byte> message, IBufferWriter<byte> signatureWriter)
```

Comment:
The `InvalidOperationException` is not documented as an exception. We should document it, along with other possible exceptions that the PAL's implementation may throw.

Reply:
Fixed